stdlib/public/stubs/FoundationHelpers.mm

```diff
@@ -61,10 +57,23 @@ static inline void initializeBridgingFunctions() {
              nullptr);
 }
 
+id
+_swift_stdlib_createTaggedPointerString(const _swift_shims_UInt8 * _Nonnull bytes,
+                                        _swift_shims_CFIndex length) {
+  initializeBridgingFunctions();
+  if (_CFStringCreateTaggedPointerString != NULL) {
+    return _CFStringCreateTaggedPointerString(bytes, length);
+  }
+  return nil;
```

Comment:
This could slightly slow down back deployed apps, since they may get NULL for this